internal/mode/static/nginx/config/servers_test.go

```diff
@@ -564,12 +633,6 @@ func TestCreateServers(t *testing.T) {
 			Value: "$connection_upgrade",
 		},
 	}
-	invalidFilterHeaderMatches := []httpMatch{
-		{
-			Headers:      []string{"filter:this"},
-			RedirectPath: "@rule10-route0",
-		},
-	}
 
 	getExpectedLocations := func(isHTTPS bool) []http.Location {
 		port := 8080
```

Comment:
if port is only used as a string val, we can define in as string (`port := "8080"`) to avoid using `strconv.Itoa(port)`